free-courses-en.md

```diff
@@ -15,6 +15,7 @@
 * [Databases](#databases)
 * [Deep Learning](#deep-learning)
 * [Docker](#docker)
+* [Flutter](#Flutter)
```

Comment:
The link didn't have to be capital. Just #flutter is fine.
Not a big deal, big figured I'd raise is since I had another comment anyway.